xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -85,6 +85,13 @@ std::string CGUIWindowPVRBase::GetSelectedItemPath(bool bRadio)
   return m_selectedItemPaths[bRadio];
 }
 
+void CGUIWindowPVRBase::ResetObservers(void)
```

Comment:
not orphaned. =>  https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/PVRManager.cpp#L662